Body/AAUHuman/LegTLEM/Seg.any

```diff
@@ -151,10 +151,10 @@ AnySeg Foot =
     
     /// For backwards Compatibility with AMMR 2.4 talus coordinate system
     AnyRefNode TalusCompatibilityFrameAMMR24 = {
-      sRel = .Scale({0.006451731, 0.02442938, ..Sign*-0.005543055});
-      ARel = {{0.9999733, -0.001715394, 0.007107884}, 
-              {0.001810773, 0.9999081, -0.01343407}, 
-              {-0.007084187, 0.01344658, 0.9998845}};
+      sRel = .Scale({0.01113461, 0.01995294, ..Sign*-0.01282196});
+      ARel = ({{0.9729412, -0.02857795, 0.2292786}, 
+              {0.01838066, 0.9987497, 0.04648894}, 
+              {-0.2303205, -0.04101672, 0.97225}}'*{{1,0,0},{0,1,0},{0,0,..Sign}})';
```

Comment:
```suggestion
      sRel = .Scale({0.01113461, 0.01995294, ..Sign*-0.01282196});
      ARel = {{0.9729412, -0.02857795, 0.2292786}, 
              {0.01838066, 0.9987497, 0.04648894}, 
              {-0.2303205, -0.04101672, 0.97225}}';
```

Reply:
This just creates a left-hand coordinate system on left leg. 

But I don't think we need to swap more axis here. The coordinate system are identical on both sides.